app-containers/docker/spec

```diff
@@ -6,11 +6,11 @@ TINI_VER=0.19.0
 
 VER=${UPSTREAM_VER}+tini${TINI_VER}
 SRCS="git::commit=tags/v${UPSTREAM_VER};rename=cli;copy-repo=true::https://github.com/docker/cli \
```

Comment:
I only see one git usage in the moby repository, consider not to copy repo for the cli repository?

Reply:
now we load version from moby repository, so only need cloen moby repository, I edited it